service/resource_router.go

```diff
@@ -34,36 +33,3 @@ func NewResourceRouter(resources map[string][]framework.Resource) func(ctx conte
 		return enabledResources, nil
 	}
 }
-
-// NewTPRResourceRouter determines which resources are enabled based upon the
-// version in the version bundle. It will be removed when we remove TPR support.
-func NewTPRResourceRouter(resources map[string][]framework.Resource) func(ctx context.Context, obj interface{}) ([]framework.Resource, error) {
```

Comment:
I do not see how the resources are now managed. There is `NewResourceRouter` but it is nowhere used, or I cannot find it in the diff. I also do not see how the resources map is created.